src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,129 @@
+import _ from 'lodash';
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		const activePanelsArr = [];
+
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			if(accordionPanel.props.isOpen) {
+				activePanelsArr.push(i);
+			}
+			return activePanelsArr;
+		});
+
+		this.state = {
+			activePanels: activePanelsArr || []
+		};
+		this._onClickTrigger = this._onClickTrigger.bind(this);
+	}
+
+	/**
+	 * Document this for PR
+	 */
+	_onClickTrigger(e, i){
+		const index = this.state.activePanels.indexOf(i);
+		const activePanelArr = this.state.activePanels;
+		const filtered = activePanelArr.filter(panel => i !== activePanelArr.indexOf(i));
+
+		if (index > -1){
+			activePanelArr.splice(index, 1);
+		} else {
+			activePanelArr.push(i);
+		}
+
+		if (!this.props.multiSelectable) {
+			_.pullAll(activePanelArr, filtered);
```

Comment:
I am strongly against importing `lodash` as a dependency - it is a very bloated library and 90% redundant with native JS array methods. In this case, I'm not exactly sure why you're pulling out the `filtered` indexes from `activePanelArr`, but I think this is the same as the more-explicit `.filter` alternative

```js
const actuallyActivePanelArr = activePanelArr.filter(panelIndex =>
  filtered.indexOf(panelIndex) === -1
);
```

Although there's some confusing mapping from `activePanelArr` to `filtered` and back into `activePanelArr`, so I'm not sure what the goal state is.